botorch_community/posteriors/bll_posterior.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import math
+
+import torch
+from torch import Tensor
+
+from botorch.posteriors import Posterior, GPyTorchPosterior
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from botorch_community.models.vblls import AbstractBLLModel
```

Comment:
is this still needed if you use `from __future__ import annotations` in all the modules to do postponed type evaluations? We should generally use that.